### ISSUE TITLE
Size of the portion of the training dataset labeled with truth labels and relation to the size of the training dataset to be labeled by Snorkel

### ISSUE THREAD

**Issue by OrranaLhaynher** [snorkel-team/snorkel-tutorials#259]
Hi! I have developed a weak labeling code based on Crowdsourcing Tutorial and I came across the following questioning: is there any test on what would be the ideal size of the ground-truth labels dataset in relation to the training dataset as a whole?

Let's say we have a 2000 line dataset. We will split the dataset into 70:30 (training and testing). How much of the 70% (training) would be a reasonable amount for having ground-truth labels? Would there be a ratio between the training dataset as a whole and the ground-truth label's part that implied greater accuracy in the labels? Is there any experiment already carried out in this regard?

Thank you very much for your attention. And any doubts about this issue, we are here to clarify! :)

**Comment by rsmith49** [comment]
Hi @OrranaLhaynher! To my knowledge, there is no explicit test already run doing what you mentioned. However, you may find [this paper](https://arxiv.org/pdf/2103.02761.pdf) interesting! It compares the added benefit of `x` labeled data points to `y` unlabeled data points in conjunction with weak supervision.
Hope that helps!

**Comment by OrranaLhaynher** [comment]
Thank you! This will help for sure!
